src/_guides/language/language-tour.md

```diff
@@ -1203,33 +1196,27 @@ The code block that follows contains the function's body:
 }; <br>
 </code>
 
-The following example defines an anonymous function with an untyped parameter, `i`.
+The following example defines an anonymous function with an untyped parameter, `item`.
 The function, invoked for each item in the list,
 prints a string that includes the value at the specified index.
 
+<?code-excerpt "test/language_tour/functions_test.dart (anonymous-function)"?>
 {% prettify dart %}
-var list = ['apples', 'oranges', 'grapes', 'bananas', 'plums'];
-list.forEach((i) {
-  print(list.indexOf(i).toString() + ': ' + i);
+var list = ['apples', 'bananas', 'oranges'];
+list.forEach((item) {
+  print('${list.indexOf(item)}: $item');
```

Comment:
IMHO we didn't need a fruit list that was that long. I also switched to using string interpolation (since the point of that example is to illustrate anonymous functions, and we'd already covered string interpolation).